nilearn/plotting/img_plotting.py

```diff
@@ -383,6 +383,17 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim='auto', black_bg='auto'):
             else:
                 black_bg = True
     if dim:
+        if dim != 'auto' and not isinstance(dim, numbers.Number):
+            raise ValueError(
+                "The input given for 'dim' needs to be "
+                "a float. "
+                "You provided dim=%s in %s" % (str(dim), type(dim)))
+        elif dim != 'auto' and dim not in range(-1, 1):
```

Comment:
I disagree with that one: to render Haby's dataset anat images I had to use dim=-1.5.